.github/workflows/deploy-experimental.yml

```diff
@@ -73,49 +73,49 @@ jobs:
           npm install
           serverless deploy --stage dev${PR}
           popd
-  deploy-text-extractor:
-    environment:
-      name: "dev"
-    runs-on: ubuntu-20.04
-    steps:
-      # Checkout the code
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          submodules: true
-      # Find the PR number.  This is not always trivial which is why this uses an existign action
-      - name: Find PR number
-        uses: jwalton/gh-find-current-pr@v1
-        id: findPr
-        with:
-          # Can be "open", "closed", or "all".  Defaults to "open".
-          state: open
-      # Configure AWS credentials for GitHub Actions
-      - name: Configure AWS credentials for GitHub Actions
-        uses: aws-actions/configure-aws-credentials@v2
-        with:
-          role-to-assume: ${{ secrets.AWS_OIDC_ROLE_TO_ASSUME }}
-          aws-region: us-east-1
-      # Deploy the text extractor lambda to AWS
-      - name: Deploy text extractor lambda
-        id: deploy-text-extractor
-        if: success() && steps.findPr.outputs.number
-        env:
-          PR: ${{ steps.findPr.outputs.pr }}
-          RUN_ID: ${{ github.run_id }}
-        run: |
-          pushd solution/text-extractor
-          npm install serverless -g
-          npm install
-          serverless deploy --stage dev${PR}
-          popd
+  # deploy-text-extractor:
+  #   environment:
+  #     name: "dev"
+  #   runs-on: ubuntu-20.04
+  #   steps:
+  #     # Checkout the code
+  #     - name: Checkout
+  #       uses: actions/checkout@v3
+  #       with:
+  #         submodules: true
+  #     # Find the PR number.  This is not always trivial which is why this uses an existign action
+  #     - name: Find PR number
+  #       uses: jwalton/gh-find-current-pr@v1
+  #       id: findPr
+  #       with:
+  #         # Can be "open", "closed", or "all".  Defaults to "open".
+  #         state: open
+  #     # Configure AWS credentials for GitHub Actions
+  #     - name: Configure AWS credentials for GitHub Actions
+  #       uses: aws-actions/configure-aws-credentials@v2
+  #       with:
+  #         role-to-assume: $#{{ secrets.AWS_OIDC_ROLE_TO_ASSUME }}
+  #         aws-region: us-east-1
+  #     # Deploy the text extractor lambda to AWS
+  #     - name: Deploy text extractor lambda
+  #       id: deploy-text-extractor
+  #       i#f: success() && steps.findPr.outputs.number
+  #       env:
+  #         PR: $#{{ steps.findPr.outputs.pr }}
+  #         RUN_ID: $#{{ github.run_id }}
+  #       run: |
+  #         pushd solution/text-extractor
+  #         npm install serverless -g
+  #         npm install
+  #         serverless deploy --stage dev${PR}
+  #         popd
   deploy-django:
     environment:
       name: "dev"
     outputs:
       url: ${{ steps.deploy-regulations-site-server.outputs.url }}
     runs-on: ubuntu-20.04
-    needs: [deploy-static, deploy-text-extractor]
+    needs: [deploy-static, ] #[deploy-static, deploy-text-extractor]
```

Comment:
I was curious if trailing commas are allowed in YAML syntax.  It appear that they are [indeed allowed](https://dev.to/this-is-learning/yaml-collections-sequences-and-mappings-4meb#:~:text=A%20flow%20sequence%20is%20a,but%20empty%20node%20entries%20disallowed.&text=or%20any%20other%20white%20space%20variation.).  Still, I don't think we use them anywhere else.